src/aws/osml/photogrammetry/elevation_model.py

```diff
@@ -24,6 +38,15 @@ def set_elevation(self, world_coordinate: GeodeticWorldCoordinate) -> None:
         :return: None
         """
 
+    @abstractmethod
+    def describe_region(self, world_coordinate: GeodeticWorldCoordinate) -> Optional[ElevationRegionSummary]:
+        """
+        Get a summary of the region near the provided world coordinate
+
+        :param world_coordinate: the coordinate at the center of the region of interest
```

Comment:
Just a small comment but I am not sure we should have a convention about these comments where we don't use "the/a/ect." to start sentences. I have also been using capitalized convention elsewhere - but it doesn't matter at much.

Reply:
I don't think there is a standard convention for this. I likely just picked up the convention I use because it was in the examples I saw on the Sphinx website: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#field-lists
 
![image](https://github.com/aws-solutions-library-samples/osml-imagery-toolkit/assets/1250221/bb515af6-cb9c-445c-9e7b-cbab6273bd17)

The "the/a" convention is used by some projects (e.g. Flask: https://flask.palletsprojects.com/en/2.3.x/api/#flask.Flask.run) and not in others (e.g. NumPy: https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html#numpy.ndarray). 

If you feel super strongly about this feel free to make a case for something the team should conform to. Until then I'd rather see the team focus on the quality / content of the documentation rather than a specific sentence structure.